### DISCUSSION TITLE
How do you see why a dxf file can not be opened?

### DISCUSSION THREAD

**Discussion by donnyv** [haplokuon/netDxf#243] [Q&A] [answered]
Is there any type of log output or toggle to see why a dxf file couldn't be open? Usually the load() just returns null with no reason why.

**Comment by haplokuon** [comment, marked as answer]
Use the debug build of the library or much better the source code. The release build will catch any exception an return null in case of an error in the DXF, the debug build will also throw assertions under some circumstances like incorrect value-code pairs, names, etc.

You can also read older post as [this](https://github.com/haplokuon/netDxf/issues/74) and [this](https://github.com/haplokuon/netDxf/issues/29).
